lib/MetaCPAN/Web/Controller/Account/Favorite.pm

```diff
@@ -26,7 +26,12 @@ sub add : Local : Args(0) {
         $res = $model->add_favorite( $data, $c->token )->recv;
     }
 
-    # Clear Fastly.. something changed
+    # TODO: validate these values?
```

Comment:
Hang over from development, they are using a standard method under the hood now, so they will be the same format - have removed the comment, thanks